zbar.yaml

```
@@ -0,0 +1,119 @@
+package:
+  name: zbar
+  version: 0.23.93
+  epoch: 0
+  description: "ZBar barcode reader library"
+  copyright:
+    - license: LGPL-2.1-or-later
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - gettext-dev
+      - glib-gir
+      - gtk-3-dev
+      - imagemagick-7
+      - imagemagick-7-dev
+      - libtool
+      - pkgconf-dev
+      - python3-dev
```

I've checked the upstream docs and cmake logs, we might need some other build-time deps:

```suggestion
      - python3-dev
      - xmlto
      - qt5-qtbase-dev
      - libx11-dev
      - libxv-dev
      - gobject-introspection-dev
      - lcms2-dev
      - docbook-xml
      - py3-supported-gobject3
      - openjdk-8-default-jdk
```